internal/image/scan.go

```diff
@@ -22,33 +29,105 @@ func ScanImage(r reporter.Reporter, imagePath string) (ScanResults, error) {
 
 	allFiles := img.LastLayer().AllFiles()
 
-	scannedLockfiles := ScanResults{
+	scanResults := ScanResults{
 		ImagePath: imagePath,
 	}
+
+	inventories := []*extractor.Inventory{}
+
 	for _, file := range allFiles {
 		if file.fileType != RegularFile {
 			continue
 		}
-		parsedLockfile, err := extractArtifactDeps(file.virtualPath, img.LastLayer())
+
+		// TODO: Currently osv-scalibr does not correctly annotate OS packages
+		// causing artifact extractors to double extract elements here.
+		// So let's skip all these directories for now.
+		// See (b/364536788)
```

Comment:
I think this bug number should be removed?

Reply:
I think this is just reference for us to keep track.